tests/frontier/precompiles/test_precompiles.py

```diff
@@ -109,3 +115,107 @@ def test_precompiles(
     post = {account: Account(storage={0: "0x00" if precompile_exists else "0x01"})}
 
     state_test(env=env, pre=pre, post=post, tx=tx)
+
+
+def precompile_addresses_in_successor(fork: Fork) -> Iterator[Tuple[str, bool]]:
+    """
+    Yield the addresses of precompiled contracts and whether they existed in the parent fork.
+
+    Args:
+        fork (Fork): The transition fork instance containing precompiled contract information.
+
+    Yields:
+        Iterator[Tuple[str, bool]]: A tuple containing the address in hexadecimal format and a
+            boolean indicating whether the address has existed in the predecessor.
+
+    """
+    predecessor_precompiles = [
+        hex(int.from_bytes(address, byteorder="big"))
+        for address in get_transition_fork_predecessor(fork).precompiles()
+    ]
+    successor_precompiles = [
+        hex(int.from_bytes(address, byteorder="big"))
+        for address in get_transition_fork_successor(fork).precompiles()
+    ]
+    for address in successor_precompiles:
+        if address in predecessor_precompiles:
+            yield address, True
+        else:
+            yield address, False
+
+
+@pytest.mark.valid_at_transition_to("Paris", subsequent_forks=True)
+@pytest.mark.parametrize_by_fork(
+    "address,exists_in_predecessor", precompile_addresses_in_successor
+)
+def test_precompile_warming(
```

Comment:
This currently only tests precompiles which are marked as precompile. So far CancunToPrague it runs up to 0x00..0011. However we also want to include addresses on top of that. For BLS, the MULG1 and MULG2 precompiles were removed, so previously the precompile set ran up to 0x00..0013. We want to test that those also are now cold (NOT warm!)

I suggest for each precompile to just check a full range, for instance up to 0x00.0020